lab_2_tokenize_by_bpe/main.py

```diff
@@ -140,6 +321,16 @@ def calculate_precision(
     :param reference: expected sequence of n-grams
     :return: value of Precision metric
     """
+    if not (
+        isinstance(actual, list) and
+        isinstance(reference, list)
+    ):
+        return None
+
+    if not actual:
+        return 0
```

Comment:
Looks like something that mypy won't like...

Reply:
Actually, I didn't get it, there no mypy errors in this string(